src/Elliptic/Systems/Xcts/BoundaryConditions/ApparentHorizon.cpp

```diff
@@ -253,6 +253,10 @@ void apparent_horizon_impl(
         make_not_null(&expansion_of_solution), make_not_null(&beta_orthogonal),
         *solution_for_negative_expansion, x, face_normal, face_normal_magnitude,
         deriv_unnormalized_face_normal);
+#ifdef SPECTRE_DEBUG
+    ASSERT(blaze::max(get(expansion_of_solution)) < 0.0,
+           "Expansion of solution is not negative everywhere.");
+#endif
```

Comment:
You don't need the ifdef.

I don't think you need the namespace `blaze::`, just `max` should work. ADL should take care of that.